demo_nodes_cpp_native/CMakeLists.txt

```
@@ -2,6 +2,12 @@ cmake_minimum_required(VERSION 3.5)
 
 project(demo_nodes_cpp_native)
 
+find_package(rmw_fastrtps_cpp QUIET)
+if(NOT rmw_fastrtps_cpp_FOUND)
+  message(STATUS "Could not find rmw_fastrtps_cpp - doing nothing")
```

This is nitpicking, but I think "skipping" is slightly clearer than "doing nothing":

```suggestion
  message(STATUS "Could not find rmw_fastrtps_cpp, skipping")
```

---

04ac472